src/plugins/discover/public/application/main/components/layout/discover_histogram_layout.test.tsx

```diff
@@ -171,58 +156,18 @@ describe('Discover main content component', () => {
       const storage = new LocalStorageMock({}) as unknown as Storage;
       const originalGet = storage.get;
       storage.get = jest.fn().mockImplementation(originalGet);
-      const component = await mountComponent({ storage });
+      await mountComponent({ storage });
       expect(storage.get).toHaveBeenCalledWith(HISTOGRAM_HEIGHT_KEY);
       expect(storage.get).toHaveReturnedWith(null);
-      expect(component.find(UnifiedHistogramLayout).prop('topPanelHeight')).toBe(undefined);
     });
 
     it('should pass the stored topPanelHeight to UnifiedHistogramLayout if a value is found in storage', async () => {
       const storage = new LocalStorageMock({}) as unknown as Storage;
       const topPanelHeight = 123;
       storage.get = jest.fn().mockImplementation(() => topPanelHeight);
-      const component = await mountComponent({ storage });
+      await mountComponent({ storage });
       expect(storage.get).toHaveBeenCalledWith(HISTOGRAM_HEIGHT_KEY);
       expect(storage.get).toHaveReturnedWith(topPanelHeight);
-      expect(component.find(UnifiedHistogramLayout).prop('topPanelHeight')).toBe(topPanelHeight);
-    });
-
-    it('should update the topPanelHeight in storage and pass the new value to UnifiedHistogramLayout when the topPanelHeight changes', async () => {
```

Comment:
Did we replace these tests? We'll need to make sure they're covered for the final PR.